cmd/fuzztool/fuzztool.go

```diff
@@ -187,21 +186,21 @@ func copyCorpus(srcRoot, dstRoot string) {
 	}
 }
 
-func main() {
-	debugF := flag.Bool("debug", false, "enable debug mode")
-	srcF := flag.String("src", "", "source, one of: 'seed', 'generated', or collected corpus' directory")
-	dstF := flag.String("dst", "", "destination, one of: 'seed', 'generated', or collected corpus' directory")
-	flag.Parse()
+//nolint:lll // no multiline field tag support
+var CLI struct {
+	Corpus struct {
+		Src string `kong:"arg,help='source, one of: \\'seed\\', \\'generated\\', or collected corpus\\' directory'"`
+		Dst string `kong:"arg,name='dst',help='destination, one of: \\'seed\\', \\'generated\\', or collected corpus\\' directory'"`
```

Comment:
Let's switch to [bare tags](https://github.com/alecthomas/kong#supported-tags) to make them more readable

Reply:
They were used before, but my motivation to use a standard go syntax was to eliminate lines like `required=""`, but sure I will revert this change